### PR TITLE
Fix null index on wa slack

### PR DESCRIPTION
**Pull request by coders-kitchen**

### CODE REVIEW

File: wa-integrations/slack/routes/management.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const Router = require('express').Router();
+const Channel = require('../models/slackChannels');
+const Contact = require('../models/contacts');
+
+const appConfig = require('../config');
+
+Router.post("/", async function(req, resp) {
+    if(appConfig.RESET_TOKEN != req.headers['x-reset-token']) {
```
Comment (Lennonka) [inline wa-integrations/slack/routes/management.js:8]: A space after "if"

Comment (coders-kitchen) [reply wa-integrations/slack/routes/management.js]: fixed :)

File: wa-integrations/slack/routes/whatsapp.js [hunk 2]
```diff
@@ -29,6 +30,7 @@ Router.route('/forwardWAMessage').post(async function (req, res) {
         {
           phone: req.body.from,
           name: req.body.whatsapp.senderName,
+          thread_ts: "no thread assigned"+req.body.from
```
Comment (Lennonka) [inline wa-integrations/slack/routes/whatsapp.js:33]: Space before and after +

Comment (coders-kitchen) [reply wa-integrations/slack/routes/whatsapp.js]: fixed :)

File: wa-integrations/slack/routes/management.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const Router = require('express').Router();
+const Channel = require('../models/slackChannels');
+const Contact = require('../models/contacts');
+
+const appConfig = require('../config');
+
+Router.post("/", async function(req, resp) {
```
Comment (Lennonka) [inline wa-integrations/slack/routes/management.js:7]: Watch use of single quote everywhere to unite the code style

Comment (coders-kitchen) [reply wa-integrations/slack/routes/management.js]: and all " replaced by '

**Pull request merged**